src/sq/semantic.md

```diff
@@ -0,0 +1,3 @@
+## Placeholder
```

Comment:
let's remove the placeholder content (and in the manifest as well) to be ready to release at any time.
we can create new pages later
and for vectors as well please